pacer.js

```diff
@@ -139,7 +139,11 @@ let PACER = {
   isSingleDocumentPage: function (url, document) {
     let inputs = document.getElementsByTagName('input');
     let lastInput = inputs.length && inputs[inputs.length - 1].value;
+    // If the receipt doesn't say "Image" we don't yet support it on the server.
+    // So far, this only appears to apply to bankruptcy claims.
+    let hasImageReceipt = !!$('td:contains(Image)').length;
```

Comment:
This is OK, but I am troubled by by the duplication of `$('td:contains(Image)').length` here with `$('td:contains(Image)').text()` in `content_delegate.js`. But the only obvious way I see to address that would be to store the value of `$('td:contains(Image)').text()` in some page state, like `this.imageString`, but that works poorly because `this` in `pacer.js` is not the same as `this` in `content_delegate.js` and there's not a great way for `PACER` functions to adjust `content_delegate` state, as far as I know. I guess `isDocumentSinglePage` could take `(url, document, that)` instead of `(url, document)` but this all gets messy and I know you're trying to cut this quickly.

So perhaps a comment here (and maybe in the other one) to note the duplication and that they need to stay in sync.